code/game/machinery/computer/arcade.dm

```diff
@@ -592,36 +609,42 @@ GLOBAL_LIST_INIT(arcade_prize_pool, list(
 		return
 	busy = TRUE
 
+	var/gamerSkill = usr.mind.get_skill_level(/datum/skill/gaming)
+	var/xp_gained = 0
 	if (href_list["continue"]) //Continue your travels
 		if(gameStatus == ORION_STATUS_NORMAL && !event && turns != 7)
 			if(turns >= ORION_TRAIL_WINTURN)
 				win(usr)
+				xp_gained += 34
 			else
 				food -= (alive+lings_aboard)*2
 				fuel -= 5
-				if(turns == 2 && prob(30))
+				if(turns == 2 && prob(30-(gamerSkill*3)))
```

Comment:
All these instances of *3 could use a define like "GAMER_SKILL_PROB_MODIFIER"

Reply:
```
#define GAMER_SKILL_PROB_MODIFIER 3
...
if(turns == 2 && prob(30 - (gamerSkill * GAMER_SKILL_PROB_MODIFIER)))```